ui/app/mainui/Popups.qml

```diff
@@ -904,9 +904,9 @@ QtObject {
             FinaliseOwnershipDeclinePopup {
                 destroyOnClose: true
 
-                onDeclineClicked:  {
+                onDeclineClicked: {
                     close()
-                    root.ownershipDeclined()
+                    root.rootStore.communityTokensStore.ownershipDeclined(communityId, communityName)
```

Comment:
root.ownershipDeclined() is caught by connection in line 892, which closed finalisePopup and then communityTokensStore.ownershipDeclined. In your version finalisePopup will not be closed.

Reply:
Hot fix in here: https://github.com/status-im/status-desktop/pull/12708/